api/src/main/java/ca/bc/gov/educ/studentdatacollection/api/repository/v1/SdcSchoolCollectionStudentRepository.java

```diff
@@ -1005,6 +1005,16 @@ WHERE stud.assignedStudentId IN (:matchedAssignedIDs)
     """)
   List<SdcSchoolCollectionStudentEntity> findAllDuplicateStudentsByCollectionID(UUID collectionID, List<UUID> matchedAssignedIDs);
 
+  @Query(value = """  
+    SELECT stud
+    FROM SdcSchoolCollectionStudentEntity stud
+    WHERE stud.assignedStudentId = :assignedStudentId
+    and stud.sdcSchoolCollection.collectionEntity.collectionID = :collectionID
+    and stud.sdcSchoolCollectionStudentID != :sdcSchoolCollectionStudentID
+    and stud.sdcSchoolCollectionStudentStatusCode != 'DELETED'
+    """)
+  List<SdcSchoolCollectionStudentEntity> findAllDuplicateStudentsInCollection(UUID collectionID, UUID assignedStudentId, UUID sdcSchoolCollectionStudentID);
```

Comment:
Suggested rephrasing for clarity - findAllDuplicatesForStudentInCollection

Reply:
Resolved.